core/local/analysis.js

```diff
@@ -180,20 +180,22 @@ function analyseEvents (events /*: LocalEvent[] */, pendingChanges /*: LocalChan
               changeFound(localChange.fileDeletion(e))
               break
             }
-            const moveChangeSamePath /*: ?LocalFileMove */ = localChange.maybeMoveFile(getChangeByPath(e))
+          }
```

Comment:
Why are we closing the block before calling `withChangeByPath`?

Reply:
It will make future diffs easier to read :stuck_out_tongue:

Reply:
OK. What was the point of using blocks for each case anyway?

Reply:
I believe vars/consts defined in different `case` branch have the same scope in js. A few of them used to have the same names, hence the necessary block statements to provide separate scope per `case` branch and prevent name clashes.